tests/phpunit/test-cache.php

```diff
@@ -904,6 +904,21 @@ public function test_wp_cache_init() {
 		$this->assertEquals( $wp_object_cache->cache, $new_blank_cache_object->cache );
 	}
 
+	public function test_redis_connect_custom_database() {
+		global $redis_server;
+		if ( ! class_exists( 'Redis' ) ) {
+			$this->markTestSkipped( 'PHPRedis extension not available.' );
```

Comment:
> No, because we want to test to make sure the object cache fails gracefully when Redis isn't available.

Ah, is that what this is doing https://github.com/pantheon-systems/wp-redis/blob/master/.travis.yml#L23 ?